pandas/plotting/_matplotlib/converter.py

```diff
@@ -473,7 +473,7 @@ def _get_default_annual_spacing(nyears) -> Tuple[int, int]:
     return (min_spacing, maj_spacing)
 
 
-def period_break(dates, period):
+def period_break(dates: PeriodIndex, period: str) -> np.ndarray:
```

Comment:
the docsting says that dates is "dates : PeriodIndex" and goes on to say "Array of intervals to monitor." could a PeriodArray be passed?

Reply:
It is only ever called with PeriodIndex.  I guess passing PeriodArray wouldnt break the world.

Reply:
while period_range returns a PeriodIndex there doesn't look like anything to gain by adding PeriodArray atm